### PR TITLE
Magicbot

### PR DESCRIPTION
**Pull request by hell0w0rld2**
Moves the robot in autonomous mode.

### CODE REVIEW

File: README.md.orig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Welcome to Robot 2020
```
Comment (tuffinmuffin) [inline README.md.orig]: bad file to add

File: autonomous/autonomous.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from magicbot import AutonomousStateMachine, tunable, timed_state
```
Comment (tuffinmuffin) [inline autonomous/autonomous.py]: Please add file description saying this is a place holder and not be used in production

File: physics.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
+#
```
Comment (tuffinmuffin) [inline physics.py]: deprecated file please do not add back in

File: requirements.txt.orig [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<<<<<<< HEAD
```
Comment (tuffinmuffin) [inline requirements.txt.orig]: Bad file, please do not add * and review each file

File: sim/config.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (tuffinmuffin) [inline sim/config.json]: depricated file please remove again

File: autonomous/autonomous.py (hunk 2 above)
Comment (ChrisHirsch) [reply autonomous/autonomous.py]: Done

**Pull request merged**